### ISSUE TITLE
Accessibility should be a goal

### ISSUE THREAD

**Issue by aron-s** [aron-s/mywebclass-agile-docs#5]
Accessibility should be a goal

_Originally posted by @morganhah99 in https://github.com/aron-s/mywebclass-agile-docs/pull/1#discussion_r1127898884_

**Comment by aron-s** [comment]
Fixed in document branch.
